style.css

```diff
@@ -0,0 +1,466 @@
+* {
+  font-family: 'lato', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.gray {
+  color: #272a31;
+}
+
+.orange {
+  color: #ec5242;
+}
+
+.white {
+  color: #d3d3d3;
+}
+
+.svg-white {
+  filter: invert(90%) sepia(0%) saturate(68%) hue-rotate(245deg) brightness(95%) contrast(93%);
+}
+
+.hide {
+  display: none;
+}
+
+h2 {
+  margin: 2rem 0;
+  padding-bottom: 1rem;
+  background-image: url('assets/images/Bar.png');
+  background-repeat: no-repeat;
+  background-size: 40px;
+  background-position-x: 50%;
+  background-position-y: 100%;
+}
+
+.btn {
+  width: 92%;
+  padding: 0.5rem;
+  border: 3px solid #ec5242;
+  color: #ec5242;
+  background-color: #fff;
+}
+
+/* Header */
+
+header {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  z-index: 1;
+}
+
+.logo {
+  display: none;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  font-size: 1.125em;
+  line-height: 24px;
+}
+
+.drawer {
+  width: auto;
+  height: 100vh;
+  background-color: rgb(236, 82, 66);
+  font-size: 32px;
+  font-weight: 700;
+  border-bottom-right-radius: 10px;
+  border-top-right-radius: 10px;
+  z-index: 1;
+}
+
+.drawer nav {
+  height: 100vh;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+.drawer ul {
+  padding: 0;
+}
+
+.drawer li {
+  margin: 0 1rem 2.5rem;
+}
+
+.menu-icon,
+.exit-icon {
+  font-size: 2rem;
+  margin: 1rem;
+}
+
+.menu a {
+  color: #272a31;
+}
+
+.event-details a {
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+.menu a:active {
+  color: #ec5242;
```

Comment:
- [ ] The hamburger menu background is supposed to be transparent or with the same color/pattern as the top section background.